### DISCUSSION TITLE
cannot import name 'slvs' from 'py_slvs'

### DISCUSSION THREAD

**Discussion by BensonTerra** [hlorus/CAD_Sketcher#364] [Announcements]
After long time trying to install the Cad Sketcher, i had sucess. But another problem appeared.

Every time i try to create something on sketch i recieve message of error, and i searched inside the folder i didnt anything what it is generating this problem. 

maybe this topic can help to understand what is happening.

![Captura de ecrã 2023-03-13 161403](https://user-images.githubusercontent.com/115477417/224762207-17341dd6-7e3d-4ff4-aab4-fa977ba756ad.png)

**Comment by FoldKjord** [comment]
Hi i think i have the same issue:
Python: Traceback (most recent call last):
  File "C:\Users\sorto\AppData\Roaming\Blender Foundation\Blender\3.4\scripts\addons\CAD_Sketcher-main\base\install_op.py", line 49, in execute
    show_package_info(slvs)
NameError: name 'slvs' is not defined

**Reply by FoldKjord** [reply]
I update Blender since 3.4 to 5.0 and it works.

**Comment by BigYingshan** [comment]
blender5.2
import slvs
